plugins/nodes/src/js/pages/nodes-overview/HostsPageContent.js

```diff
@@ -154,6 +156,33 @@ class HostsPageContent extends React.Component {
       </div>
     );
   }
+  getGridViewResourceSwitch() {
+    if (!this.props.location.pathname.includes("grid")) {
+      // we have to return an empty div here (instead of null),
+      // because FilterBar has fixed rightAlignLastNChildren={2}
```

Comment:
Would it be possible to extract the `this.props.location.pathname.includes("grid")` into a function so that we don't have to use a fixed value?